src/hockeypuck/conflux/recon/settings_test.go

```diff
@@ -216,6 +216,18 @@ partners=["1.2.3.4:11370","5.6.7.8:11370"]
 	}
 }
 
+func (s *SettingsSuite) TestAddFilters(c *gc.C) {
```

Comment:
Suggest adding a test case that intentionally adds duplicate filter strings. Duplicates at the beginning, in the middle, and at the end post-sort.

Reply:
fixed in 9efd916